docs/source/getstart.rst

```diff
@@ -0,0 +1,227 @@
+          "epoch": 1,
+          "main/accuracy": 0.9421835541725159,
+          "validation/main/accuracy": 0.9703000783920288
+      }, 
+      {
+          "main/loss": 0.07222291827201843,
+          "validation/main/loss": 0.08141259849071503,
+          "iteration": 1200,
+          "elapsed_time": 19.54666304588318,
+          "epoch": 2,
+          "main/accuracy": 0.9771820902824402,
+          "validation/main/accuracy": 0.975399911403656
+      },
+      ...
+  ]
+
+without ``Trainer`` code example, a  brief extract take from `MNIST custom loop example <https://github.com/chainer/chainerui/blob/master/examples/train_mnist_custom_loop.py>`__:
+
+.. code-block:: python
+
+  from chainerui.utils import LogReport
+
+  def main():
+
+      # [ChainerUI] setup log reporter to show on ChainerUI along with 'args'
+      ui_report = LogReport(args.out, conditions=args)
+      while train_iter.epoch < args.epoch:
+
+          # ...train calculation
+
+          if train_iter.is_new_epoch:
+
+              # [ChainerUI] write values to 'log' file
+              stats = {
+                  'epoch': train_iter.epoch,
+                  'iteration': train_iter.epoch * args.batchsize,
+                  'train/loss': train_loss, 'train/accuracy': train_accuracy,
+                  'test/loss': test_loss, 'test/accuracy': test_accuracy
+                  }
+              ui_report(stats)
+
+.. note::
+
+   :ref:`chainerui's LogReport <module_log_report>` only sets ``exampled_time``, so train loop has to set ``epoch`` and ``iteration`` manually.
+
+Experimental conditions
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: ../images/result_table_condition_sample.png
+
+ChainerUI shows training job with experimental conditions read from ``args`` file. ``args`` file is a JSON file, which includes key-value pairs. See :ref:`save_args <module_save_args>`, util function to dump command line arguments or dictionary to ``args`` file.
+
+Setup example, a brief extract taken from `MNIST example <https://github.com/chainer/chainerui/blob/master/examples/train_mnist.py>`__:
+
+.. code-block:: python
+
+  # [ChainerUI] import chainerui util function
+  from chainerui.utils import save_args
+
+  def main():
+      parser.add_argument('--out', '-o', default='result',
+                          help='Directory to output the result')
+      args = parser.parse_args()
+
+      # [ChainerUI] save 'args' to show experimental conditions
+      save_args(args, args.out)
+
+``args`` file example, values are showed as experimental conditions on a results table::
+
+  {
+      "resume": "",
+      "batchsize": 100,
+      "epoch": 20,
+      "frequency": -1,
+      "gpu": 0,
+      "unit": 1000,
+      "out": "results"
+  }
+
+Operate training loop
+~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: ../images/detail_page_operation_block.png
+
+ChainerUI support to operate a training loop with :ref:`CommandsExtension <module_command_extension>`, the latest version supports:
```

Comment:
ChainerUI supports operating a training loop with CommandsExtension. The latest version supports: